### PR TITLE
FEC-11250 Exoplayer upgrade v2.14.0

### PR DESCRIPTION
**Pull request by GouravSna**

### CODE REVIEW

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java [hunk 1]
```diff
@@ -352,25 +347,25 @@ public void onCues(List<Cue> cues) {
         }
 
         @Override
-        public void onVideoSizeChanged(int width, int height, int unappliedRotationDegrees, float pixelWidthHeightRatio) {
+        public void onVideoSizeChanged(VideoSize videoSize) {
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java]: @GouravSna  please add null check

File: playkit/src/main/java/com/kaltura/playkit/profiler/ExoPlayerProfilingListener.java [hunk 2]
```diff
@@ -404,12 +418,12 @@ public void onDroppedVideoFrames(EventTime eventTime, int droppedFrames, long el
     }
 
     @Override
-    public void onVideoSizeChanged(EventTime eventTime, int width, int height, int unappliedRotationDegrees, float pixelWidthHeightRatio) {
-        log("VideoSizeChanged", field("width", width), field("height", height));
+    public void onVideoSizeChanged(EventTime eventTime, VideoSize videoSize) {
+        log("VideoSizeChanged", field("width", videoSize.width), field("height", videoSize.height));
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/profiler/ExoPlayerProfilingListener.java:422]: @GouravSna null check here as well

File: playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java (hunk 1 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java]: Marked as NonNull as inside ExoPlayer, it is not null

File: playkit/src/main/java/com/kaltura/playkit/profiler/ExoPlayerProfilingListener.java (hunk 2 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/profiler/ExoPlayerProfilingListener.java:422]: Done

**Review by giladna — APPROVED** [review]

**Pull request merged**